components/competition-add/AddCompetitionFormStepDetails.vue

```diff
@@ -151,15 +151,15 @@ export default defineComponent({
           @change="context.uploadFile({
             changeEvent: $event,
           })"
-          @load="context.releaseImageObjectUrl({
-            objectUrl: context.imageSourceRef.value,
-          })"
         >
 
         <img
           :src="context.generateCompetitionImageUrl()"
           alt="League badge"
           class="image"
+          @load="context.releaseImageObjectUrl({
+            objectUrl: context.imageSourceRef.value,
```

Comment:
@hecticme 

Since `context` has already known `#imageSourceRef`, does not require to pass the parameter.

It is OK to kick out the parameter from `#releaseImageObjectUrl()` in this pull request

```suggestion
            objectUrl: context.imageSourceRef.value,
```